docs/static/monitoring-apis.asciidoc

```diff
@@ -266,55 +303,55 @@ Example response:
 [source,js]
 --------------------------------------------------
 {
-  "jvm": {
-    "threads": {
-      "count": 35,
-      "peak_count": 36
+  "jvm" : {
+    "threads" : {
+      "count" : 49,
+      "peak_count" : 50
     },
-    "mem": {
-      "heap_used_in_bytes": 318691184,
-      "heap_used_percent": 15,
-      "heap_committed_in_bytes": 519045120,
-      "heap_max_in_bytes": 2075918336,
-      "non_heap_used_in_bytes": 189382304,
-      "non_heap_committed_in_bytes": 200728576,
-      "pools": {
-        "survivor": {
-          "peak_used_in_bytes": 8912896,
-          "used_in_bytes": 9538656,
-          "peak_max_in_bytes": 35782656,
-          "max_in_bytes": 71565312,
-          "committed_in_bytes": 17825792
+    "mem" : {
```

Comment:
@jsvd I'm going to leave the changes as they are here. I agree that whitespace is minor, but I also think it's good for the docs to match what the software actually returns, and in this case, the extra space was introduced in the formatting. I try to update the examples whenever I test the APIs because it helps make the doc look less dated over time...sorry if it cause you a little extra time to confirm the changes.